raft-tech/konfirm
# Feature/helm

## What's Included
This introduces a standard Helm chart for deploying Konfirm and some helper commands in the Makefile.

## How to test
`make helm-install-kind`
`make kind-push-mock`
`kubectl apply -f ./config/samples/`
Verify the sample Konfirm testsuite is created and properly runs konfirm tests.
